mesop/web/src/editor/editor.ng.html

```diff
@@ -1,4 +1,4 @@
-<mat-sidenav-container class="container">
+<mat-sidenav-container class="sidenav-container">
```

Comment:
I'm curious, was this causing a naming conflict with bootstrap?

Reply:
Yes, they have a class named "container". So when I first tried loading the bootstrap.css, the layout was messed up and eventually tracked it down to this usage.